Content/Scan QR/scanQR.json

```diff
@@ -0,0 +1,10 @@
+{
+	"scanQR_connectorExtension_instructions": "Scan the QR code in the Radix Connector browser extension.",
+	"scanQR_connectorExtension_disclosureTitle": "Don't have the Radix Connector browser extension?",
+	"scanQR_connectorExtension_disclosureItem1": "Go to **wallet.radixdlt.com** in your desktop browser.",
+	"scanQR_connectorExtension_disclosureItem2": "Follow the instructions there to install the Radix Connector.",
+
+	"scanQR_account_instructions": "Scan a QR code of a Radix Account address from another wallet or an exchange.",
+	
+	"scanQR_importOlympia_instructions": "Scan the QR code shown in the Export section of the Radix Desktop Wallet for Olympia."
```

Comment:
These two are already present [here](https://github.com/radixdlt/apps-localization-src/blob/main/Content/1.0%20-%20Asset%20Transfer/assetTransfer.json#L4) & [here](https://github.com/radixdlt/apps-localization-src/blob/main/Content/0.6%20-%20Import%20Olympia%20Accounts/importOlympiaAccounts.json#L3).

I placed them under this new file so the corresponding view can determine its content by a given context (`connectorExtension`, `account` or `importOlympia`). This way, we set the view's context/type, and not what instructions to show, whether to show the disclosure items at the bottom or not, what should those items say, etc.